app/models/transaction.rb

```diff
@@ -0,0 +1,22 @@
+class Transaction < ApplicationRecord
+  belongs_to :user
+
+  validates :amount, :date, presence: true
+  validates :amount, numericality: { greater_than: 0, less_than: 999999999 }
+
+  def amount=(value)
+    write_attribute(:amount, return_amount_with_two_decimal(value)) if value
+  end
+
+  private
+
+  def return_amount_with_two_decimal(value)
+    if value.to_s.include?('.')
+      num = value.to_s.split('.')
+      num[1] = num[1][0..1]
```

Comment:
Logic here is to remove all the extra numbers and leave only two numbers after point;
Also, remove auto rounding, so when we trying to pass ```2.129``` for example, we will get ```2.12```, instead of ```2.13```

Reply:
Hrm, interesting. I mean all that description of logic. Why don't comment with a code? Let's a create a service for that:
```ruby
class UserMoneyValue
  def initialize(value_in_string)
    @value_in_string = value_in_string
  end

  def with_two_numbers_after_point
    if has_point?
      ...
    else
      value_in_string.to_i
    end
  end

  private

  attr_reader :value_in_string

  def has_point?
    value_in_string.include?('.')
  end

  ...
end
```

Reply:
@atatarenko Can I create this service and tests for it in this PR? https://github.com/wazzuper/Cashy/pull/20
It's easier to testing it with interface